ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml

```diff
@@ -818,7 +846,6 @@ tests:
     cluster_profile: packet-assisted
     dependencies:
       HYPERSHIFT_IMAGE: hypershift-operator
-      PROVIDER_IMAGE: cluster-api-provider-agent
```

Comment:
why did you remove it?
it's especially important to have it before next FF to make sure we're not mistakenly using the wrong branch of capi-provider-agent